### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by adnanaman**

### CODE REVIEW

**Comment by hugo53** [comment]
@adnanaman We have a typo mistake here 
```
+- [http://remotus.com/](http://remotus.com/) a fulltime jobs for developers, still under construction. Seems have a lots of
 +- [http://www.candoerz.com/](http://www.candoerz.com/) hourly update remote, relocate, freelance jobs aggregator. 
 +- startup jobs.
```
Could you please edit and resend PR, or I will add candoerz.com directly to the list? Thanks for contributing, mate!

File: README.md [hunk 1]
```diff
@@ -46,7 +46,9 @@ Awesome Remote Work
 - [https://www.wfh.io/](https://www.wfh.io/), 100% remote tech-oriented job board
 - [http://jobs.remotive.io/](http://jobs.remotive.io/), weekly update with remote jobs from startups.
 - [http://remoteok.io/](http://remoteok.io/) a remote job aggregator. Job from Angel List, Stackoverflow, ...
-- [http://remotus.com/](http://remotus.com/) a fulltime jobs for developers, still under construction. Seems have a lots of startup jobs.
+- [http://remotus.com/](http://remotus.com/) a fulltime jobs for developers, still under construction. Seems have a lots of
+- [http://www.candoerz.com/](http://www.candoerz.com/) hourly update remote, relocate, freelance jobs aggregator. 
+- startup jobs.
```
Comment (hugo53) [inline README.md]: @adnanaman Could you please edit and resend PR, or I will add candoerz.com directly to the list? Thanks for contributing, mate!

Comment (adnanaman) [inline README.md]: thanks @hugo53  please add it as appropriate.

Comment (hugo53) [inline README.md]: Ok let me do @adnanaman . Thank you in advance!

**Pull request closed without merging**